features/smamodbus.py

```diff
@@ -374,25 +374,33 @@
         9314: 'PlugwiseCircle',
         9315: 'PlugwiseSting',
         9316: 'SCS-1000',
-        9317: 'SB 5400TL-JP-22'
+        9317: 'SB 5400TL-JP-22',
+        9324: 'SBS1.5-1VL-10',
+        9325: 'SBS2.0-1VL-10',
+        9326: 'SBS2.5-1VL-10',
+        9344: 'STP4.0-3AV-40',
+        9345: 'STP5.0-3AV-40',
+        9346: 'STP6.0-3AV-40',
+        9366: 'STP3.0-3AV-40'
+    },
+    'BatteryState': {
```

Comment:
I am not quite sure this will work or i can not find the function: AFAIK the values here are return values that are present in Modbus Register: `30955` and cannot be called directly.

Reply:
Correct, but these values are not used to call modbus directly. They are used to interpret the modbus response codes here: https://github.com/datenschuft/SMA-EM/blob/cdf88e0a931d01a411f465bfa0abc4b727cba7ff/features/smamodbus.py#L459